index.js

```diff
@@ -1,14 +1,15 @@
 const routes = require('./src/routes.js')
 require('dotenv').config()
+require('./models/movie')
 const express = require('express')
-
-const app = express()
+const app  = express()
 const port = process.env.PORT
 app.set('view engine', 'pug')
 app.set('views', './views')
 app.use(express.static('./public'))  // store front-end files in ./public
 
 routes(app)
 
+
 app.listen(port, () => console.log(`App listening on port ${port}`))
 
```

Comment:
this file doesn't end up having any changes, try to reset it to the master version.